src/lib/util/generateId/index.test.ts

```diff
@@ -0,0 +1,10 @@
+import generateId from '.';
+
+describe('generateId', () => {
+  it('should return two different ids', () => {
+    const firstId = generateId();
+    const secondId = generateId();
+
+    expect(firstId).not.toEqual(secondId);
+  });
```

Comment:
Not sure if it'd be worth it to write regression tests for the format `xxxx-xxxx-xxx-xxxx` and/or length of the string? (only if it's relevant)

Reply:
You regression as a way to make sure it's compatible with previous usages?